src/Loggers/OwnJSONPatternFormatter.cpp

```diff
@@ -62,6 +69,26 @@ void OwnJSONPatternFormatter::formatExtended(const DB::ExtendedLogMessage & msg_
     const Poco::Message & msg = msg_ext.base;
     DB::writeChar('{', wb);
 
+    if (!date_time_utc.empty())
+    {
+        writeJSONString(date_time_utc, wb, settings);
+        DB::writeChar(':', wb);
+
+        DB::writeChar('\"', wb);
+        static const DateLUTImpl & utc_time_zone = DateLUT::instance("UTC");
+        /// Change delimiters in date for compatibility with old logs.
```

Comment:
Could you please clarify - change delimiters from what to what? And compatibility with which old logs?